components/RichText.vue

```diff
@@ -2,7 +2,7 @@
 <template>
   <div
     v-if="content"
-    class="rich-text mx-auto mt-14 md:p-6"
+    class="rich-text text-black mt-14 md:p-6"
```

Comment:
This will cause an issue in the Blog Post page if we are removing the auto margins and it will no longer center the blog text @erikaaquino